### PR TITLE
Big refactoring

### LINKED ISSUE #28: Add new method to Subscriptions Query Builder
there are some new resources related to subscriptions

GET to kladama.com/subsc/status/0  should list all PAUSED subscriptions
GET to kladama.com/subsc/status/1   should list all ACTIVE subscriptions

GET to kladama.com/subsc/user/dev/status/0  should list all PAUSED subscriptions of user Dev
GET to kladama.com/subsc/user/dev/status/1   should list all ACTIVE subscriptions of user Dev

The query builder needs a "**by_status**" method, as it already has a "**by_user**" one.

### PR DESCRIPTION
**Pull request by hdkrus**
closes #28

### CODE REVIEW

File: tests/integration_test.py [hunk 1]
```diff
@@ -598,232 +603,347 @@ def test_get_phenomena_by_name():
...
+        print('\n')
+
+    def test_around(self):
         # given
-        user = 'ensoag'
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).last_n(5)
+        session = _get_dev_session()
+        query = Query().subsc.by_user(self._user).filter_by(self._subscription).results.around(5, '20200519', '20200622')
```
Comment (wasdal) [inline tests/integration_test.py:768]: change "**filter_by**" to "**by_key**"  ??

Comment (hdkrus) [reply tests/integration_test.py]: Addressed

File: tests/integration_test.py [hunk 2]
```diff
@@ -588,242 +593,373 @@ def test_get_phenomena_by_name():
...
 
-    @staticmethod
-    def test_last_n():
         # given
-        user = 'ensoag'
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).last_n(5)
+        session = _get_dev_session()
+        query = Query().subsc.by_user(self._user).by_key(self._subscription).with_dates('20200519', '20200622')
```
Comment (wasdal) [inline tests/integration_test.py:785]: shouldn't it be  "...by_key(self._subscription).**dates.period**('20200519', '20200622')"  ????  idem to results.period

File: tests/integration_test.py [hunk 3]
```diff
@@ -588,242 +593,373 @@ def test_get_phenomena_by_name():
...
 
-    @staticmethod
-    def test_last():
         # given
-        user = 'ensoag'
-        subscription = '3JSM4MN89SJFLE7VR6KPCM0BVPXTWT'
-        session = _get_sandbox_session()
-        query = Query().subsc.by_user(user).filter_by(subscription).last
+        session = _get_dev_session()
+        query = Query().subsc.by_user(self._user).by_key(self._subscription).get_dates()
```
Comment (wasdal) [inline tests/integration_test.py:770]: shouldn't it be "Query().subsc.by_user(self._user).by_key(self._subscription).**dates**"   ???

File: tests/integration_test.py (hunk 2 above)
Comment (hdkrus) [reply tests/integration_test.py:785]: lo mismo para la url entonces no?

File: tests/integration_test.py (hunk 3 above)
Comment (hdkrus) [reply tests/integration_test.py:770]: ok...working on it

**Pull request merged**